neon/data-collection/data-format/index.md

```diff
@@ -165,7 +165,7 @@ This file contains data recorded by the integrated [IMU](/data-collection/data-s
 | **recording id** | Unique identifier of the recording this sample belongs to.     |
 | **timestamp [ns]** | UTC timestamp in nanoseconds of the sample.    |
 | **gyro x [deg/s]**<br />**gyro y [deg/s]**<br />**gyro z [deg/s]** | Rotation speed around x, y or z-axis respectively in degrees/s.    |
-| **acceleration&nbsp;x&nbsp;[G]**<br />**acceleration&nbsp;y&nbsp;[G]**<br />**acceleration z [G]** | Translational acceleration along the x, y or z-axis respectively in G. Note `1 G = 9.80665 m/s^2`.|
+| **acceleration&nbsp;x&nbsp;[g]**<br />**acceleration&nbsp;y&nbsp;[g]**<br />**acceleration z [g]** | Translational acceleration along the x, y or z-axis respectively in units of [g-force](https://en.m.wikipedia.org/wiki/G-force). Note `1 g-force = 9.80665 m/s^2`.|
```

Comment:
Technically, `g-force` is the name of the force and `g` is the name of the unit, isn't it? Using `g-force` as the unit in the text seems inaccurate. 

If we want to mention g-force explicitly we could maybe say "Translational acceleration/g-force along..."? Or we just keep the link as is but change `g-force` to `g`.

Reply:
'g-force' is a confusing name, I find. It is force per unit mass, so it works out to a unit of acceleration. Some texts clarify that it should not be considered a force.

The unit of 'g-force' is written as 'g', but that is also easily confused with the 'g' of free-fall acceleration, which points downwards (provided you use a standard reference frame), whereas, when you stand on Earth, 'g-force' points upwards. 'G-force' is technically derived from 'g', so the absolute magnitude is the same, hence 'g' as the unit, but the direction is different.

True, you cannot really say "units of g-force", but rather, "total g-force that an object experiences". It is "the number of g units of acceleration." I was trying to be compact and not add too much text. I will think of something better, as I feel it is useful for users that just read this part quickly and don't go through the link. For example, when you lay Neon down and upright on a table, you get +g for acceleration Z, and when turn Neon on its right side, you get -g for X acceleration. Both of which are confusing if you assume the 'g' of free-fall acceleration.

Reply:
How about:

Translational acceleration (in terms of [g-force](https://en.m.wikipedia.org/wiki/G-force)) along the x, y or z-axis respectively. Note `1 g = 9.80665 m/s^2`.

Or maybe "acceleration (measured as g-force)"?

Reply:
Either version seems fine to me! :+1:

Reply:
@marc-tonsen Ok, done. Thanks for catching that!